pkgs/applications/display-managers/greetd/default.nix

```diff
@@ -8,13 +8,13 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "greetd";
-  version = "0.9.0";
+  version = "0.10.0";
 
   src = fetchFromSourcehut {
     owner = "~kennylevinsen";
     repo = pname;
     rev = version;
-    sha256 = "sha256-w68iKabVVIHB+ujCd5q+L4T6+Tfqedou3wXOuMGi6aw=";
+    sha256 = "sha256-QFxE5c4XDOBa3rKpUFur6xOG+5m0NSCOWo7960ZWEow=";
   };
 
   cargoHash = "sha256-vn1ih4YXsltR1zdy7BzS3CSJO9Uewx/654lRTSkWV0k=";
```

Comment:
This probably needs to be updated

Reply:
How do you get the cargoHash?

Reply:
It does

```
error: hash mismatch in fixed-output derivation '/nix/store/5fci6drfrdpby4gd48chdbxp5159f66w-greetd-0.10.0-vendor.tar.gz.drv':
         specified: sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=
            got:    sha256-w7V5ZBFy5MIEQKOPPgsc7XnrXsF5rEDNzUKeLPPy4P4=
```

Reply:
You give it a fake hash (just leave it blank), try to build it, then wait for the new one

Reply:
I can set cargoHash to empty in a new github pull request, but I'm wondering where in the build process on github do you get that error message for the new hash?

Reply:
You run `nix-build -A greetd` locally in your copy of the nixpkgs repo to test your changes and bulids.

Reply:
It has to be inside your `nixpkgs` repo, you can't run it from an arbitrary point

I would suggest reading a guide on contributing to nixpkgs first - you can start with pkgs/README.md and CONTRIBUTING.md